tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -225,6 +224,16 @@ def __init__(self, *args, reuse_window=False, **kwargs):
         super().__init__(*args, **kwargs)
         self.protocol.reuse_window = reuse_window
 
+    def convert_result(self, test, result, extra=None):
+        test_result, subtest_results = super().convert_result(test, result, extra)
+        # Chromium CI should note when there are compiled expectations that
+        # never ran. See web-platform/tests#43499 for details.
+        subtest_names = {subtest.name for subtest in subtest_results}
+        for unknown_subtest in sorted(test.subtests - subtest_names):
+            subtest_results.append(
+                test.subtest_result_cls(unknown_subtest, "NOTRUN", message=None))
```

Comment:
If unknown subtests get a "NOTRUN" result, what should rebaseline tool behave?

Reply:
Is it possible we do something in the 'WPTResultsProcessor'? If there are unknown subtests, we simply flag that test as unexpectedly failed (if it is not a crash/timeout previously).

We do not need to do anything in Wptrunner, especially we do not need to retry such tests.

Reply:
This is usually an unexpected wptrunner `NOTRUN`, which is [mapped to](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/tools/blinkpy/w3c/wpt_results_processor.py;l=68;drc=bb4791908e272d17263b154fb830990203b7c058;bpv=0;bpt=0) ResultDB [unexpected failure](https://ci.chromium.org/ui/p/chromium/builders/try/linux-rel/1678090/overview) that the rebaseline tool downloads.

An expected `NOTRUN` like:

```
This is a testharness.js-based test.
[NOTRUN] does not exist
```

won't cause a failure by itself. Ideally, we wouldn't check subtests anyway during a timeout, so maybe this isn't important to clean up.

> Is it possible we do something in the 'WPTResultsProcessor'?

We can try, since `WPTResultsProcessor` can handle the above expected `NOTRUN` case. `WPTResultsProcessor` needs to manipulate the `wpt run` exit code too then.